src/algorithms/tracking/AmbiguitySolverConfig.h

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+// Copyright (C) 2024 Minjung Kim
+
+#pragma once
+
+#include <vector>
+
+namespace eicrecon {
+struct AmbiguitySolverConfig {
+  /// Maximum amount of shared hits per track.
+  std::uint32_t maximum_shared_hits = 1;
+  /// Maximum number of iterations
+  std::uint32_t maximum_iterations = 100000;
+  /// Minimum number of measurement to form a track.
+  std::size_t n_measurements_min = 3;
```

Comment:
Judging by report https://github.com/eic/EICrecon/pull/1383#issuecomment-2060110240, this introduces a significant loss in efficiency, even for the truth seeding. I'm wondering if there is a set of parameters that would be less aggressive?

Reply:
Actually, looking at the truth seeded tracks again, I notice that those track that are gone, while numerous, don't have enough points in the first place:
<img width="400" alt="Screen Shot 2024-05-22 at 12 24 39 PM" src="https://github.com/eic/EICrecon/assets/245573/c87f181e-2957-4b17-8131-a34ddb1b1fbe">
So this might be fine.